flink/connectors/kafka.rst

```diff
@@ -0,0 +1,28 @@
+===============
+Kafka Connector
+===============
+
+Due to its nature, the distributed publish/subscribe Apache Kafka is one of the best matches to read data from and write data to for Flink.
+
+The `official documentation <https://ci.apache.org/projects/flink/flink-docs-release-1.0/apis/streaming/connectors/kafka.html>`_ contains in-depth details on the high level of interoperability between Kafka and Flink.
+
+To use the connector, you should first of all include the following dependency in your project's POM:
+
+.. literalinclude:: /rbp-docs-code/mvn/pom.xml
```

Comment:
This is how the pox gets rendered on the branch.
![schermata 2016-05-17 alle 10 15 21](https://cloud.githubusercontent.com/assets/7675964/15315556/97bac422-1c18-11e6-91ca-9c5bbb6fe5fe.png)